### ISSUE TITLE
Unable to load defaultValue for enum args

### ISSUE THREAD

**Issue by cbandy** [rmosolgo/graphql-ruby#592]
I have a schema with the following arg:

```json
{
  "name": "orderBy",
  "description": "The method to use when ordering `Role`.",
  "type": {
    "kind": "ENUM",
    "name": "RolesOrderBy",
    "ofType": null
  },
  "defaultValue": "PRIMARY_KEY_ASC"
}
```

When trying to load it, I get the following error:

```
#<JSON::ParserError: 743: unexpected token at 'PRIMARY_KEY_ASC'>
/home/cbandy/.gem/ruby/2.1.2/gems/json-1.8.6/lib/json/common.rb:155:in `parse'
/home/cbandy/.gem/ruby/2.1.2/gems/json-1.8.6/lib/json/common.rb:155:in `parse'
/home/cbandy/.gem/ruby/2.1.2/gems/graphql-1.4.4/lib/graphql/schema/loader.rb:110:in `define_type'
/home/cbandy/.gem/ruby/2.1.2/gems/graphql-1.4.4/lib/graphql/schema/loader.rb:105:in `block in define_type'
/home/cbandy/.gem/ruby/2.1.2/gems/graphql-1.4.4/lib/graphql/schema/loader.rb:104:in `map'
/home/cbandy/.gem/ruby/2.1.2/gems/graphql-1.4.4/lib/graphql/schema/loader.rb:104:in `define_type'
/home/cbandy/.gem/ruby/2.1.2/gems/graphql-1.4.4/lib/graphql/schema/loader.rb:96:in `block in define_type'
/home/cbandy/.gem/ruby/2.1.2/gems/graphql-1.4.4/lib/graphql/schema/loader.rb:95:in `map'
/home/cbandy/.gem/ruby/2.1.2/gems/graphql-1.4.4/lib/graphql/schema/loader.rb:95:in `define_type'
/home/cbandy/.gem/ruby/2.1.2/gems/graphql-1.4.4/lib/graphql/schema/loader.rb:23:in `block in load'
/home/cbandy/.gem/ruby/2.1.2/gems/graphql-1.4.4/lib/graphql/schema/loader.rb:21:in `each'
/home/cbandy/.gem/ruby/2.1.2/gems/graphql-1.4.4/lib/graphql/schema/loader.rb:21:in `load'
```

I had trouble finding examples of an enum default value, but from what I can tell the schema is correct. (For example, GraphiQL is happy.)

When I dump the test schema with the following change, I see this:

```diff
diff --git a/spec/graphql/schema/loader_spec.rb b/spec/graphql/schema/loader_spec.rb
index d4d7373d..11fe139b 100644
--- a/spec/graphql/schema/loader_spec.rb
+++ b/spec/graphql/schema/loader_spec.rb
@@ -103,6 +103,7 @@ describe GraphQL::Schema::Loader do
         argument :id, !types.ID
         argument :varied, variant_input_type, default_value: { id: "123", int: 234, float: 2.3, enum: :foo, sub: [{ string: "str" }] }
         argument :variedWithNull, variant_input_type_with_nulls, default_value: { id: nil, int: nil, float: nil, enum: nil, sub: nil, bigint: nil, bool: nil }
+        argument :enum, choice_type, default_value: :foo
       end
 
       field :content do
```

```json
{
  "name": "enum",
  "description": null,
  "type": {
    "kind": "ENUM",
    "name": "Choice",
    "ofType": null
  },
  "defaultValue": "\"FOO\""
}
```

**Comment by rmosolgo** [comment]
Hmm, I wonder if that's a bug on our side. Which language did you use to generate your initial schema JSON?

**Comment by cbandy** [comment]
This schema is coming out of [postgraphql](https://github.com/calebmer/postgraphql). It's using the [`graphql`](https://www.npmjs.com/package/graphql) Node.js package.

**Comment by rmosolgo** [comment]
Taking a try in #605

**Comment by rmosolgo** [comment]
This will be fixed in 1.5.2 which I'll release today or tomorrow!
